tests/UnitTests/Catalog/Services/CommentsRepository/AddCommentTests.cs

```diff
@@ -70,5 +70,29 @@ public void NegativeCommentsProductId_ThrowsValidationException()
             Assert.AreEqual(result.Status, TaskStatus.Faulted);
             Assert.IsInstanceOfType(result.Exception.InnerException, typeof(ValidationException));
         }
+
+        //TODO: Repair
```

Comment:
This comment is alarming me: what should be repaired? Either remove the comment or make necessary changes and remove comment as well.